frontend/src/functions/formatPhone.ts

```diff
@@ -1,39 +1,31 @@
-import {
-  formatPhoneNumber,
-  getCountryCallingCode,
-  parsePhoneNumber,
-} from "react-phone-number-input";
-/**
- * E.164 number as a string.
- * Example +13337891234
- * E.164 is an international numbering plan for public telephone systems in which each assigned number
- * contains a country code (CC), a national destination code (NDC), and a subscriber number (SN).
- * There can be up to 15 digits in an E.164 number.
- */
-export type PhoneString = string;
-
 /**
  * Make sure that when we format phone numbers in a consistent way.
+ * phoneNumber: The phone number to format.
+ * withCountryCode: Whether to include the country code in the formatted number.
  */
-export function formatPhone(phoneNumber: PhoneString) {
-  const parsedNumber = parsePhoneNumber(phoneNumber);
+export function formatPhone(
+  phoneNumber: string,
+  withCountryCode = false
```

Comment:
Not too big of an issue with a single function, but a good habit to get into: positional optional parameters can be hard to read at the calling site if you're not familiar with the function, and also make refactoring more work if you want to add additional optional arguments later.  You can avoid such problems by adding an `options` parameter, so that it is called like `formatPhone("+1 (250) 555 - 1234", { withCountryCode: true })` instead of `formatPhone("+1 (250) 555 - 1234", true)`:

```suggestion
  options: { withCountryCode?: boolean } = {}
```